src/zfs-generator.c

```diff
@@ -319,7 +319,9 @@ int generateSysrootUnit(char *directory, int bootfs, char *dataset, char *snapsh
 [Mount]\n\
 What=%s\n\
 Type=initrd_zfs\n\
-Options=%s\n", what, options);
+Options=%s\n\
```

Comment:
This doesn't belong into the `Mount` section, but into the `Unit` section that I usually place before the `Mount` section.

Reply:
Woops, wasn't paying attention.